modules/javafx.graphics/src/main/native-glass/win/GlassClipboard.cpp

```diff
@@ -29,6 +29,12 @@
 #include <hash_map>
 #include <hash_set>
 
+// The following is derived from _HASH_SEED, which is an internal constant from
+// include/xhash; since this is an internal constant we define our own rather
+// that relying on something that Microsoft could remove (and apparately has in
+// newer version of Visual C++)
+#define GLASS_HASH_SEED (size_t)0xdeadbeef
```

Comment:
Is there any logic behind this seemingly weird value? And why is it defined in hex as opposed to, say, decimal 3735928559?

Reply:
Is it some magical number that makes hash bucket collisions less frequent or something?

Reply:
I'm not sure why they chose this number, but rather than trying to figure out whether we rely on its value, it seems a safer change to leave it the same as the existing constant.

Reply:
All I can think of is it happens to have a good, seemingly random, distribution of 1s and 0s: "11011110101011011011111011101111". Anyways, off-topic, just kind of got me thinking.